### PR TITLE
docs: intro page

### PR DESCRIPTION
**Pull request by gazpachu**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @tablecheck/tablekit-input-button@1.0.2-canary.29.c6fc41e987b16076066d4531f4d9a2dedcc298a5.0
  npm install @tablecheck/tablekit-modal-dialog@1.0.2-canary.29.c6fc41e987b16076066d4531f4d9a2dedcc298a5.0
  # or 
  yarn add @tablecheck/tablekit-input-button@1.0.2-canary.29.c6fc41e987b16076066d4531f4d9a2dedcc298a5.0
  yarn add @tablecheck/tablekit-modal-dialog@1.0.2-canary.29.c6fc41e987b16076066d4531f4d9a2dedcc298a5.0
  ```
</details>

### CODE REVIEW

File: pages/Intro/Intro.stories.mdx [hunk 1]
```diff
@@ -9,14 +9,14 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 ![Tablekit](https://cdn3.tablecheck.com/common/images/tablekit/v1.0.0/hero.svg)
 
-![pipeline status](https://gitlab.com/tablecheck/frontend/tablekit/badges/master/pipeline.svg)
-![coverage status](https://gitlab.com/tablecheck/frontend/tablekit/badges/master/coverage.svg)
+![pipeline status](https://github.com/tablecheck/tablekit/badges/master/pipeline.svg)
+![coverage status](https://github.com/tablecheck/tablekit/badges/master/coverage.svg)
```
Comment (SimeonC) [inline pages/Intro/Intro.stories.mdx:13]: Neither of these exist and can be removed.

**Review by SimeonC — CHANGES_REQUESTED** [review]
This is not a fix - it's documentation, use the label `internal` and change the commit to "docs: ...`

File: pages/Intro/Intro.stories.mdx (hunk 1 above)
Comment (gazpachu) [reply pages/Intro/Intro.stories.mdx]: fixed

**Comment by gazpachu** [comment]
@SimeonC this should be ready to merge

**Review by SimeonC — APPROVED** [review]

**Review by daniellizik — APPROVED** [review]

**Review by staevs — APPROVED** [review]

**Pull request merged**